_docs/release-schedule.md

```diff
@@ -5,34 +5,19 @@ order: 8
 longtitle: What to expect in future releases of Portus
 ---
 
-## Release schedule for 2.3
+## Release schedule for 2.4
 
-The 2.3 release will be mostly about fixing known bugs and testing. This release
-will also improve the documentation on how to deploy and use Portus in
-production, while taking special care of the official Docker image. This has
-already been done (see [this PR](https://github.com/SUSE/Portus/pull/1254)), but
-we will improve the situation even further until the final release.
+We plan to have a shorter development cycle for the 2.4 release. This is because
+for the 2.3 release we spent too much time without doing an actual release, so
```

Comment:
` That's because we spent too much time without doing an actual release and users ...`